third_party/gpus/rocm_configure.bzl

```diff
@@ -561,69 +573,14 @@ def _create_local_rocm_repository(repository_ctx):
             out_dir = "rocm/include",
             exceptions = ["gtest", "gmock"],
         ),
-        make_copy_dir_rule(
```

Comment:
In our internal ROCM build, we now get failures which seem to point to removing these rules.
We still run with ROCM 5.0, maybe these rules are still needed for that system?

Reply:
Some of the backward compatibility for ROCm5.0 was missing from this PR. Sorry for the troubles. Can you try again?

Reply:
Thanks, I will give it another try :) It might take some time though because I am out of office next week, and not sure whether it will go through all approvals today. But maybe someone else will make sure this gets merged while I am away.

Reply:
And it is merged :) Thanks for making it backwards compatible with ROCM 5.0, this made it much easier. Now I can move our internal build to ROCM 5.3 without having to do it at the same time.